turbo/src/lib/mock_data/csv/csv_no_video.json

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "interview": "Alice",
+        "comment-body": "The weather is indeed pleasant today",
+        "comment-id": "0"
+    },
+    {
+        "interview": "Bob",
+        "comment-body": "When will the rain stop.",
+        "comment-id": "28"
+    }
+]
```

Comment:
missing line at the end of the file.

Reply:
Not sure what you mean!

Reply:
<img width="362" alt="Screenshot 2024-01-12 at 1 35 44 p m" src="https://github.com/AIObjectives/talk-to-the-city-reports/assets/57231302/912f5451-d692-4b24-a794-2b5d5e6997cf">
this little symbol indicates a missing empty line at the end of the file, which normally all files should have.

Reply:
Ah you mean empty line?